concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1047	1047	zurückhalten	zurückhalten	to restrain		
+Steinthal-1875-1565-1048	1048	zurückgeben	zurückgeben	to return something		
+Steinthal-1875-1565-1049	1049	bringen	bringen	to bring	692	BRING
+Steinthal-1875-1565-1050	1050	holen	holen	to get	694	GET
+Steinthal-1875-1565-1051	1051	verlieren	verlieren	to lose	1660	LOSE
+Steinthal-1875-1565-1052	1052	hergeben	hergeben	to hand something over		
+Steinthal-1875-1565-1053	1053	verweigern	verweigern	to refuse	49	REFUSE
+Steinthal-1875-1565-1054	1054	hinlegen	hinlegen	to put something down	998	PUT
+Steinthal-1875-1565-1055	1055	hinstellen	hinstellen	to put something down	998	PUT
+Steinthal-1875-1565-1056	1056	heben	heben	to lift	501	LIFT
+Steinthal-1875-1565-1057	1057	etwas aufheben (aufnehmen)	etwas aufheben (aufnehmen)	to pick up something	371	PICK UP
+Steinthal-1875-1565-1058	1058	ablegen	ablegen	to cast off		
+Steinthal-1875-1565-1059	1059	gehen	gehen	to walk	1443	WALK
+Steinthal-1875-1565-1060	1060	treten	treten	to kick	1135	KICK
+Steinthal-1875-1565-1061	1061	stehen	stehen	to stand	1442	STAND
+Steinthal-1875-1565-1062	1062	sitzen	sitzen	to sit	1416	SIT
+Steinthal-1875-1565-1063	1063	sich setzen	sich setzen	to sit down	1649	SIT DOWN
+Steinthal-1875-1565-1064	1064	liegen	liegen	to lie	1411	LIE (REST)
+Steinthal-1875-1565-1065	1065	sich legen	sich legen	to lie down	215	LIE DOWN
+Steinthal-1875-1565-1066	1066	laufen	laufen	to run	1519	RUN
+Steinthal-1875-1565-1067	1067	knien	knien	to kneel	66	KNEEL
+Steinthal-1875-1565-1068	1068	kriechen	kriechen	to crawl	3456	CRAWL OR CREEP
+Steinthal-1875-1565-1069	1069	springen	springen	to jump	1829	JUMP
+Steinthal-1875-1565-1070	1070	sich anlehnen	sich anlehnen	to lean on someone or something		
+Steinthal-1875-1565-1071	1071	hängen	hängen	to hang	2884	HANG
+Steinthal-1875-1565-1072	1072	sinken	sinken	to sink	1088	SINK (DESCEND)
+Steinthal-1875-1565-1073	1073	fallen	fallen	to fall	1280	FALL
+Steinthal-1875-1565-1074	1074	sich neigen	sich neigen	to tilt		
+Steinthal-1875-1565-1075	1075	bücken	bücken	to bend down	2761	BEND (ONESELF)
+Steinthal-1875-1565-1076	1076	kommen	kommen	to come	1446	COME
+Steinthal-1875-1565-1077	1077	weggehn	weggehn	to go away	1752	LEAVE
+Steinthal-1875-1565-1078	1078	hereinkommen	hereinkommen	to come inside		
+Steinthal-1875-1565-1079	1079	hinausgehn	hinausgehn	to go outside	1107	GO OUT
+Steinthal-1875-1565-1080	1080	hineingehen	hineingehen	to go inside		
+Steinthal-1875-1565-1081	1081	spazieren gehen	spazieren gehen	to stroll	2890	WALK (TAKE A WALK)
+Steinthal-1875-1565-1082	1082	zurückkehren	zurückkehren	to return	581	COME BACK
+Steinthal-1875-1565-1083	1083	wohnen (zu Hause sein)	wohnen (zu Hause sein)	to reside	1099	RESIDE
+Steinthal-1875-1565-1084	1084	wohnen (zu Hause sein)	zu Hause sein	to be at home		
+Steinthal-1875-1565-1085	1085	Reise	Reise	journey	2393	JOURNEY
+Steinthal-1875-1565-1086	1086	Gepäck	Gepäck	baggage		
+Steinthal-1875-1565-1087	1087	Stab	Stab	cane	1296	WALKING STICK
```

Comment:
Double mapping

Reply:
Since the other (Steinthal-1875-1565-174) is a walking stick proper I'd unmap this.